metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/DefaultAuthorizationService.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.common.server.usermetadata;
+
+import com.netflix.metacat.common.QualifiedName;
+import com.netflix.metacat.common.exception.MetacatException;
+import com.netflix.metacat.common.server.properties.Config;
+import com.netflix.servo.util.VisibleForTesting;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Config based authorization Service implementation.
+ *
+ * @author zhenl
+ * @since 1.2.0
+ */
+public class DefaultAuthorizationService implements AuthorizationService {
+    //catalog+database name as the acl control key, and userNames as the value
+    private final Map<String, Set<String>> createACL;
+    private final Map<String, Set<String>> deleteACL;
+
+    /**
+     * Constructor.
+     *
+     * @param config metacat config
+     */
+    public DefaultAuthorizationService(
+        final Config config
+    ) {
+        this.createACL = this.getACLMap(config.getMetacatCreateAcl());
+        this.deleteACL = this.getACLMap(config.getMetacatDeleteAcl());
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean isUnauthorized(final String userName,
```

Comment:
We should check if metacat.authorization.enabled=true. If it is false, then it should be a no-op.